.github/workflows/conventional-commit.yml

```
@@ -28,23 +28,18 @@ jobs:
-          echo 'EOF' >> $GITHUB_ENV
-
       - uses: amannn/action-semantic-pull-request@v5
         id: lint_pr_title
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          requireScope: true
-          scopes: |
-            ${{ env.SCOPES }}
-
+        requireScope: true
```

actionlint

unexpected key "requireScope" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)